src/util/Constants.js

```diff
@@ -644,6 +670,21 @@ exports.WebhookTypes = [
   'Channel Follower',
 ];
 
+/**
+ * The type for sticker format
+ * * PNG
+ * * APNG
+ * * LOTTIE
+ * @typedef {string} StickerFormatTypes
+ */
+exports.StickerFormatTypes = [
```

Comment:
this should be added to the typings for the constants

Reply:
do you mean `index.d.ts`?

Reply:
yes